modules/base/scale/nonuniformstaticscale.cpp

```diff
@@ -36,6 +36,16 @@ namespace {
         openspace::properties::Property::Visibility::NoviceUser
     };
 
+    // This Scale type scales the scene graph node that it is attached to by a fixed
+    // amount that does not change over time. It is possible to change the fixed scale
+    // after starting the application, but it otherwise remains unchanged. The scaling is
+    // a simple multiplication so that a `Scale` value of 10 means that the object will be
+    // 10 times larger than it should be. In comparison to the StaticScale type, this type
+    // has the ability to scale an object by different amounts for each direction.
+    //
+    // This type can be used to adjust the aspect ratio of certain Renderable types, for
+    // example to make a RenderableSphericalGrid that is not a perfect spherical grid, but
+    // a tri-axial ellipsoid instead.
     struct [[codegen::Dictionary(NonUniformStaticScale)]] Parameters {
         // [[codegen::verbatim(ScaleInfo.description)]]
         glm::dvec3 scale;
```

Comment:
Is this a positive value only?

Reply:
There might be reasons to have a negative scale, in case someone wants to mirror a model for example.